Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+@Library('github.com/cloudogu/ces-build-lib@1.48.0')
+import com.cloudogu.ces.cesbuildlib.*
+
+node('docker') {
+
+    def git = new Git(this)
+    def mvn = new MavenWrapperInDocker(this, 'azul/zulu-openjdk-alpine:11.0.10')
+
+    timestamps {
+        catchError {
+            timeout(activity: false, time: 60, unit: 'MINUTES') {
+
+
+                stage('Checkout') {
+                    git branch: 'main', url: 'https://github.com/cloudogu/gitops-playground'
+                    dir('etcd') {
+                        checkout scm
+                    }
+                }
+
+                yamllintImage = "cytopia/kubeval:0.15"
+
+                stage("Lint k8s Resources") {
+                    new Docker(this)
+                            .image(yamllintImage)
+                            .inside("-v ${WORKSPACE}/etcd/manifests/:/data -t --entrypoint=")
+                                    {
+                                        sh "kubeval etcd/manifests/etcd.yaml --ignore-missing-schemas"
+                                    }
+                }
+
+                stage('Start gitops playground') {
+                    clusterName = createClusterName()
+                    startK3d(clusterName)
+                    String registryPort = sh(
+                            script: 'docker inspect ' +
+                                    '--format=\'{{ with (index .NetworkSettings.Ports "30000/tcp") }}{{ (index . 0).HostPort }}{{ end }}\' ' +
+                                    " k3d-${clusterName}-server-0",
+                            returnStdout: true
+                    ).trim()
+                    sh("sudo snap install kubectl --classic")
```

Comment:
A specified version of `kubectl` could be helpful for a deterministic build.

Reply:
This is kind of hard to achieve via snap, so I'll keep it like that for now.